src/Cli/dotnet/Commands/CliCommandStrings.resx

```diff
@@ -2483,4 +2483,7 @@ To display a value, specify the corresponding command-line option without provid
   <data name="ZeroTestsRan" xml:space="preserve">
     <value>Zero tests ran</value>
   </data>
-</root>
+  <data name="SolutionAddReferencedProjectsOptionDescription" xml:space="preserve">
+    <value>Recursively add projects' ReferencedProjects to solution</value>
```

Comment:
Referring to ReferencedProjects is clear to you but probably opaque to most people

Reply:
"Recursively add project references to the solution"?